src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String name, int radius, String color) {
+        super(name, color);
+        this.radius = radius;
+        setArea(radius);
+    }
+
+    public String drawFigure() {
```

Comment:
lets create interface `FigureDrawer` and add one abstract method 'drawFigure()'

Reply:
I crate intarface FigureDrawer.